modules/display.c

```diff
@@ -365,6 +367,37 @@ static gboolean backlight_ioctl(int value)
 	return status;
 }
 
+/* Set DPMS mode for X11 display to on or off
+ *
+ * @params dpms_on To force the mode into on or off (blanked) mode.
+ */
+static void display_dpms(gboolean dpms_on)
+{
+	Display* dpy = NULL;
+
+	dpy = XOpenDisplay(NULL);
+	if (dpy == NULL) {
+		dpy = XOpenDisplay(":0.0");
+	}
+
+	if (dpy == NULL) {
+		mce_log(LL_CRIT, "display_dpms: unable to open display");
```

Comment:
This one would be better:
`mce_log(LL_CRIT, "%s: unable to open display", __func__);`

Reply:
And I'd change the log level to LL_INFO.
It's because a user may want to run mce without Xorg running.

Reply:
I think X11 not running on Maemo Leste is pretty critical, no?

Reply:
1. I don't see any problems in my setup with mce running and Xorg still not configured (and not running).
2. It maybe a good idea to allow a user to choose, if he wants to boot in text mode.
3. Running Maemo Leste without Xorg might be good for debugging.
4. And it's just fun. Something like `mplayer -vo aa` or `mplayer -vo caca` may appear on youtube.

Reply:
All of that will work just fine when MCE logs as `LL_CRIT`, though?

Reply:
Sure. But there's no need to pollute logs without a need.
If Xorg is not running, it's not something critical for mce.

Reply:
if this fails because DISPLAY is not provided, or for another reason, the device will likely lose power way faster than normal, so I think it's probably critical for proper operation of the device. If you want it to not be visible I can turn it into info.

Reply:
Thanks.
I think the user will notice that Xorg is not running without mce's hints :-)